src/scheduler/VertexScheduler/VertexExecutionHistory.tsx

```diff
@@ -0,0 +1,303 @@
+/**
+ * @license
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React, { useEffect, useState } from 'react';
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
+import { PickersDayProps, PickersDay } from '@mui/x-date-pickers/PickersDay';
+import { DateCalendar } from '@mui/x-date-pickers/DateCalendar';
+import { Box, LinearProgress } from '@mui/material';
+import dayjs, { Dayjs } from 'dayjs';
+
+import { authApi, handleDebounce } from '../../utils/utils';
+import VertexJobRuns from './VertexJobRuns';
+import VertexJobTaskLogs from './VertexJobTaskLogs';
+import { IconLeftArrow } from '../../utils/icons';
+
+interface IDagRunList {
```

Comment:
Move this to common file and use it where ever required. This interface is repeated.